README.md

```diff
@@ -169,3 +172,17 @@ For `dsub` server documentation, see [servers/dsub](servers/dsub/README.md).
 
 ## Job Manager `cromwell` Server
 For `cromwell` server documentation, see [servers/cromwell](servers/cromwell/README.md).
+
+## Build docker images and releases
+From v0.2.0, Job Manager starts to release stock docker images on [DockerHub](https://hub.docker.com/u/databiosphere)
```

Comment:
Not really a comment, but are we planning on publishing every version release to DockerHub?

Reply:
Maybe @cjllanwarne here. @bfcrampton I don't think we have reached any team agreements on this, at least for Cromwell, this might be the plan(I'm not sure).

Reply:
Since as I understand it our main release artifact is a set of docker containers (and hopefully eventually a startup/shutdown script to orchestrate them together), it makes sense to me that every release would be pushed to a docker repo.

I'm not *especially* wed to Dockerhub in particular but I do think it should be cloud-agnostic and dockerhub has the benefit of being the hegemon right now. And Cromwell is on dockerhub too which also makes it a nice default choice.